__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+const app = require("../app");
+const db = require("../db/connection");
+const request = require("supertest");
+
+beforeEach(() => seed(testData));
+
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  test("should respond with an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        expect(res.body.length).toBe(3);
+      });
+  });
+  test("should respond with correct properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        const body = res.body;
+        body.forEach((topic) => {
+          expect(topic).toHaveProperty("slug");
+          expect(topic).toHaveProperty("description");
+        });
+      });
+  });
+  test("should respond with status code 404 if the endpoint is not found", () => {
+    return request(app).get("/api/invalid-endpoint").expect(404);
+  });
```

Comment:
A good test to consider, but deserves it's own describe block mentioning error handling because it's doing something new